HW2/P4/P4.txt

```diff
@@ -0,0 +1,7 @@
+number of threads=1, 2.54070901871 seconds for 10 filter passes.
+number of threads=2, 1.27839994431 seconds for 10 filter passes.
+number of threads=4, 0.679213047028 seconds for 10 filter passes.
+
```

Comment:
For 1 thread, I got roughly 4.3 seconds, which is similar to not applying parallelism.